rtf_vis_tool/src/Components/gtsfm_edge_list.js

```diff
@@ -0,0 +1,58 @@
+//List of all Edges in GTSFM Visualization Graph
+//imported in DivGraph.js
```

Comment:
was DivGraph renamed?

Reply:
yup. forgot to update the comment. I've updated it now.